browser/resources/settings/brave_system_page/brave_system_page_index.html

```diff
@@ -32,12 +32,10 @@
   <settings-memory-page slot="view" id="memory" prefs="{{prefs}}" in-search-mode="[[inSearchMode_]]">
   </settings-memory-page>
 
-  <template is="dom-if" if="[[showBatterySettings_]]">
```

Comment:
I think this probably still needs to be in a template dom-if or the system page might show up in search results for battery when the section is hidden